### ISSUE TITLE
Make tools in gallery thumbnails always visible

### ISSUE THREAD

**Issue by basilleaf** [SETI/rms-opus#216]
no hover rquired

**Comment by basilleaf** [comment]
Note that people are using screenshots of this page in other media/art (ie _The Furthest_) I think we still need the gallery tools to not require mouse hover even on desktop for accessibility, but perhaps this should be user configurable like a "hide/show tools" checkbox.

**Comment by rfrenchseti** [comment]
Something like this is required for OPUS to be usable on a mobile device.

**Comment by rfrenchseti** [comment]
Actually this works fine on mobile devices so is no longer required.
